### ISSUE TITLE
Why not use common.deduct_event_handler_class here?

### ISSUE THREAD

**Issue by mrvanes** [SURFscz/SRAMsync#23]
https://github.com/SURFscz/SRAMsync/blob/b9f2904276a22884027a90c9c0f99652ab6bcd44/SRAMsync/cua_script_generator.py#L102
common.py has deduct_event_handler_class, why not reuse that here?

**Comment by venekamp** [comment]
No longer relevant due to new implementation of running EventHandlers in parallel.
